test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1

```diff
@@ -220,33 +215,28 @@ Describe 'Classes methods with inheritance' -Tags "CI" {
     Context 'Method calls' {
 
         It 'can call instance method on base class' {
-            class bar
-            {
-                [int]foo() {return 100500}
+            class bar {
+                [int]foo() { return 100500 }
```

Comment:
@mawosoft Please revert all style changes. If you want you can open new PR for style changes as our guideline requires and we will merge that PR before current one.